Build/CMake/Modules/AtomicCommon.cmake

```diff
@@ -203,3 +203,19 @@ macro (setup_executable)
     setup_target ()
 
 endmacro ()
+
+macro (include_directories_exported)
+    include_directories(${ARGV})
```

Comment:
I thought about these names a bit, I think "_exported" is pretty accurate, it would be good to add some comments about what these macros do here